### PR TITLE
instant search: enable v2 with waffle flag

### PR DESCRIPTION
**Pull request by LeoMcA**
upgrade django-waffle

https://github.com/mozilla/sumo-project/issues/753

### CODE REVIEW

File: kitsune/sumo/jinja2/base.html [hunk 1]
```diff
@@ -327,6 +327,8 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
   <script src="{{ static('jsi18n/{lang}/djangojs.js'|f(lang=request.LANGUAGE_CODE|lower)) }}"></script>
 {% endif %}
 
+<script src="{{ url("wafflejs") }}"></script>
```
Comment (akatsoulas) [inline kitsune/sumo/jinja2/base.html:330]: How about using waffle to control whether we are going to load this?

**Review by akatsoulas — APPROVED** [review]
r+wc

File: kitsune/sumo/jinja2/base.html (hunk 1 above)
Comment (akatsoulas) [reply kitsune/sumo/jinja2/base.html:330]: This is a nit feel free to ignore. This will be cached but for the use case that we have it shouldn't matter.

Comment (LeoMcA) [reply kitsune/sumo/jinja2/base.html:330]: I'll follow up with some ideas I had to avoid the caching problem in another PR

**Pull request merged**